include/deal.II/lac/vector.templates.h

```diff
@@ -1472,6 +1472,14 @@ Vector<Number>::lp_norm (const real_type p) const
 }
 
 
```

Comment:
please use three spaces before and after this function to be consistent with the general convention.